src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1767,8 +1767,15 @@ def run(self):
     # If yes, bail out.
     logs.log('Checking for bad build.')
     crash_revision = environment.get_value('APP_REVISION')
-    is_bad_build = testcase_manager.check_for_bad_build(self.job_type,
-                                                        crash_revision)
+
+    (is_bad_build, should_ignore_crash_result,
+     build_run_console_output) = testcase_manager.check_for_bad_build(
```

Comment:
It might be time to introduce a small class to hold these - three return values feels like a lot.

Edit: maybe you intend to define a proto message to hold these? That would make sense to me.

Reply:
+1, let's go with a namedtuple.

Reply:
ended up using a dataclass since I wanted to use the type from tests also.